pkg/migrations/sql/idempotent/base.sql

```diff
@@ -1627,6 +1629,13 @@ BEGIN
     SELECT older_than + INTERVAL '1 hour'
     INTO check_time;
 
+    startT := clock_timestamp();
+
+    PERFORM set_config('application_name', format('promscale maintenance: data retention: metric %s', metric_name), false);
+    IF log_verbose THEN
+        RAISE LOG 'promscale maintenance: data retention: metric %: starting', metric_name;
```

Comment:
If `log_verbose` is true, we should also log the `startT` here, maybe at the start of the log. This is in order to know the absolute time when the maintenance started.

Reply:
Logs include the timestamp when the logging occurs in Postgres. There is no need to log it in the message.